### ISSUE TITLE
Generate my weekly standup report

### ISSUE THREAD

**Issue by vdavez** [18F/standup-bot#56]
As a team member, I want standup bot to let me generate a report for a given time period of all of my standups, so that I can use them when I report in Tock.

Scenario: Weekly report for standup bot
Given I am a user of standup bot with username @vdavez
And I have reported standups for a period between 1/1/16 and 1/6/16 
When I DM "@standup-bot report 2016-01-01 2016-01-06"
Then standup-bot should reply:

```
Standup report for @vdavez for 2016-01-01 to 2016-01-06:
2016-01-01: 
Y: Blah
T: Blah Blah
B: Stuff
G: Other stuff
2016-01-02:
Y: ...
***
```

**Comment by stvnrlly** [comment]
I thought a little about how this command should work. Here's what I'm thinking:

`report`: get the last 5 business days for yourself
`report @user`: get the last 5 business days for `@user`
`report 3`: get the last 3 business days for yourself
`report #channel`: get today's report for `#channel`
`report #channel 2016-5-3`: get the report for `#channel` from `2016-5-3`

There are probably some other combinations available there. Am I missing other use cases?

**Comment by mgwalker** [comment]
Closely related to #65?

**Comment by stvnrlly** [comment]
Yes! This is that.

**Comment by mgwalker** [comment]
Since #65 is merged, I think this is closed?

**Comment by stvnrlly** [comment]
We still don't have date ranges, so I'm not sure that it should be closed yet.

**Comment by vdavez** [comment]
New acceptance criteria, and with this change, the issue is closed.

```
Scenario: Weekly report for standup bot
Given I am a user of standup bot with username @vdavez
And I have reported standups for a period between 1/1/16 and 1/6/16 
When I DM "@standup-bot report vdavez 3"
Then standup-bot should reply:

Summary
Days reported 
3

2016-01-01: 
Y: Blah
T: Blah Blah
B: Stuff
G: Other stuff
2016-01-02:
Y: ...
***
```

**Comment by vdavez** [comment]
Implemented in #65.
